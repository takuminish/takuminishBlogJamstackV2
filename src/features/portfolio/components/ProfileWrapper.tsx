import Box from "@mui/material/Box";
import { ReactNode } from "react";

type Props = {
  children: ReactNode;
};

function ProfileWrapper(props: Props) {
  const { children } = props;

  return <Box sx={{ margin: "16px 0" }}>{children}</Box>;
}

export default ProfileWrapper;
