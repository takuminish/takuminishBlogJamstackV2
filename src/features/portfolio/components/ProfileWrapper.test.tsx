import React from "react";
import { render } from "@testing-library/react";
import ProfileWrapper from "./ProfileWrapper";

const createDOM = () => {
  return render(<ProfileWrapper>test</ProfileWrapper>);
};

test("snapshot", () => {
  const { container } = createDOM();

  expect(container).toMatchSnapshot();
});
