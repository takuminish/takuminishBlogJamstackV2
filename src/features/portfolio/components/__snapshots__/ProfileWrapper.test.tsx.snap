// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-1jbawm2"
  >
    test
  </div>
</div>
`;
